Leanplum/Manager/LPRequestQueue.m

```diff
@@ -75,6 +80,13 @@ - (void)sendRequests:(void(^)(void))success failure:(void(^)(NSError *error))fai
                 }
             }
         }
+        if (requests.count <= maxCount) {
+            [[LPRequestQueue sharedInstance] sendRequests:^{
```

Comment:
Why do we need to call this ?

Reply:
I have udpated naming for ease of reading. Its for edge case. If the total requests in batch are more than 10,000 (max requests). Then we make another batch api call. We basically want to make sure the queue is empty.

Reply:
Ah i see, the `[LPRequestManager deleteRequestsWithLimit:maxCount];` was confusing me